Automattic/wp-calypso
# Stats Widget: Add empty state placeholder and styling

Related to #76379 

## Proposed Changes

* Update empty state text and styling.
* Add placeholder data and styling for the empty chart.

<img width="599" alt="截圖 2023-05-02 下午11 06 22" src="https://user-images.githubusercontent.com/6869813/235712267-c401d4ac-df90-4676-b106-c08631fa996b.png">

## Testing Instructions



* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php`.
* Ensure the empty chart section aligns with the latest empty state design.

## Pre-merge Checklist
